mozjs-sys/build.rs

```diff
@@ -56,11 +47,7 @@ const SM_TARGET_ENV_VARS: &'static [&'static str] = &[
     "OBJCOPY",
 ];
 
-const EXTRA_FILES: &'static [&'static str] = &["makefile.cargo", "src/jsapi.cpp"];
```

Comment:
I think we only need makefile.cargo as cc should emit `cargo:rerun-if-changed` for us.